### ISSUE TITLE
compatibility flags may be incorrect

### ISSUE THREAD

**Issue by haydenm315** [dmnsgn/media-codecs#4]
I'm working from h265 bitstreams which bares some similarities to your work here.  I don't think your'e correct in the compatibility value being between 0..32.  
compatibility flags are a 32-bit hexadecimal value, where each bit indicates some level of profile compatibility. Each bit set indicates some specific profile that the stream has compatibility with. Can be a value between 0 and FFFFFFFF.    I'm referencing your below comment in the code.

https://github.com/dmnsgn/media-codecs/blob/eb06e09162dd7e2106d87978f9db6937a27d2e06/src/hevc.js#L32

**Comment by dmnsgn** [comment]
You are right. That kind of make the possible list of codec strings blow up so I'll need to update my example too.

If you have any source to share on the spec for compatibility (and constraints), I am curious.
